flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/IcebergTableSource.java

```diff
@@ -17,14 +17,18 @@
  * under the License.
  */
 
-package org.apache.iceberg.flink;
+package org.apache.iceberg.flink.source;
```

Comment:
moved this class inside the `source` dir so that it can use some package private methods from `IcebergSource`. and it also seems like a good home for this class.